.circleci/config.yml

```diff
@@ -4,62 +4,97 @@
 #
 version: 2
 jobs:
-  build:
+  mysql:
     docker:
       # specify the version you desire here
-      - image: circleci/ruby:2.5.1-node-browsers
-
-      # Specify service dependencies here if necessary
-      # CircleCI maintains a library of pre-built images
-      # documented at https://circleci.com/docs/2.0/circleci-images/
+      - image: circleci/ruby:2.5-node-browsers
       - image: circleci/mysql:5.7
-      - image: circleci/postgres:9.4
```

Comment:
If we are only going to test with one database, I think I would prefer that we use the one we run with in production.

Reply:
now using the same one

Reply:
I saw that we're using the 10.7 version, I updated it to use that version